app/api/srch/typeahead.rb

```diff
@@ -0,0 +1,124 @@
+require 'grape'
+require 'grape-entity'
+
+module Srch
+  class Typeahead < Grape::API
+
+    # Endpoint definitions
+    resource :typeahead do
+
+      # Request URL should be /api/typeahead/all?srchString=QRY&seq=KEYCOUNT
+      # Basic implementation from classic plots2 SearchController
```

Comment:
The Swagger doc endpoint (and the whole Swagger UI) is used to interpret this, though I've been having trouble with mine--the definitions section of the Swagger doc output is not describing the output classes completely, and I'm still trying to figure out the problem.

In the meantime, I was going to write up a how-to research note (actually, several notes) describing usage of these endpoints.  That would include description of the output and basic/advanced work with the included javascript code.